src/styles.module.css

```diff
@@ -161,7 +161,11 @@ form input {
 }
 
 .progress_btn {
-  width: 40px;
+  width: 60px;
+}
+
+.completed_stat {
+  font-size: 34px;
 }
 
 .current_chapter_title {
```

Comment:
- [ ] Great job so far @sulaimonaa There are just a few changes I will suggest you implement and you'll be on your way to the  next Milestone :+1: 
- [ ] I have mentioned some style fixes that need to be made in the screenshot below. So kindly fix them so your design looks exactly like the Given design :wink: 
- Your Design
![image](https://github.com/sulaimonaa/book-store/assets/102699958/958f736a-5159-4654-b5ad-fcabe3aff813)

- Zeplin Design
![image](https://github.com/sulaimonaa/book-store/assets/102699958/bf6a53fc-d2d3-4b15-b884-026a0123e688)